.azure/pipelines/jobs/default-build.yml

```diff
@@ -57,7 +56,7 @@ parameters:
   buildDirectory: ''
 
 jobs:
-- job: ${{ coalesce(parameters.jobName, parameters.agentOs) }}
+- job: ${{ parameters.jobName }}
```

Comment:
Revert this change; it provides a fallback when the parameter isn't provided and doesn't hurt. May be needed in other pipelines for this branch.

Reply:
Pushed a commit to my test branch that reverts this, will update this branch with the same change once I'm done testing